### DISCUSSION TITLE
Choose editor to open on breakpoint hit

### DISCUSSION THREAD

**Discussion by apupier** [microsoft/vscode-discussions#616] [Extension Development QnA] [answered]
is it possible to specify which editor to be opened when a breakpoint is hit? or maybe to specify that an editor doesn't support `debug` functionality?

In my use case, I would like to have the textual editor to be opened instead of a custom graphical editor which doesn't support debug.

**Comment by isidorn** [comment, marked as answer]
Currently not possible. You as the debugger provide a URI and VS Code will open that URI in the way it thinks is best.
